test/tei_to_es_test.rb

```diff
@@ -42,17 +42,16 @@ def test_fields
     assert_equal "1865-07-01", whitman["date_not_before"]
     # source
     # note that source has been replaced by has_source 
-    # TeiToES at this point does not define default behavior for this field
-    # source = "Track and stable talk, Aberdeen, South Dakota, 1888-02-24"
-    # assert_equal source, cody["has_source"]
+    source = {"title"=>"Track and stable talk, Aberdeen, South Dakota, 1888-02-24"}
+    assert_equal source, cody["has_source"]
 
-    # source = "Neihardt, John Gneisenau, 1881-1973, Letter from John G. Neihardt to Julius T. House, 1927-11-05"
-    # assert_equal source, neihardt["has_source"]
+    source = {"title"=>"Neihardt, John Gneisenau, 1881-1973, Letter from John G. Neihardt to Julius T. House, 1927-11-05"}
+    assert_equal source, neihardt["has_source"]
 
     # # NOTE: this document does have publisher information in the biblStruct and will
     # # need to override the default xpaths to obtain that information for the source field
-    # source = "Walt Whitman, Walt Whitman to a Soldier, late 1865 (?)"
-    # assert_equal source, whitman["has_source"]
+    source = {"title"=>"Walt Whitman, Walt Whitman to a Soldier, late 1865 (?)"}
+    assert_equal source, whitman["has_source"]
```

Comment:
I think we should change all the `source` comments and variables to also say `has_source` now too. And then the comment `# note that source has been replaced by has_source` can be removed too